lib/libutee/include/tee_internal_api.h

```diff
@@ -243,7 +243,10 @@ TEE_Result TEE_CloseAndDeletePersistentObject1(TEE_ObjectHandle object);
 
 TEE_Result TEE_RenamePersistentObject(TEE_ObjectHandle object,
 				      const void *newObjectID,
-				      uint32_t newObjectIDLen);
+				      size_t newObjectIDLen);
+TEE_Result __GP11_TEE_RenamePersistentObject(TEE_ObjectHandle object,
+					     const void *newObjectID,
+					     uint32_t newObjectIDLen);
```

Comment:
For `TEE_InitRefAttribute()` you did not introduce a `_GP11_` function, but here you do. Why?

Reply:
Because it was already introduced in "GP131: Update TEE_Attribute"